js/mobile-menu.js

```diff
@@ -0,0 +1,17 @@
+const hamburger = document.querySelector('.hamburger-icon');
+const mobileNav = document.querySelector('.mobile-navigation');
+const defaultNav = document.querySelector('.navigation-menu');
+const xButton = document.querySelector('.img-x');
+
+function displayMobileMenu() {
+  defaultNav.style.display = 'none';
+  mobileNav.style.display = 'unset';
+}
+
+function hideMobileMenu() {
+  mobileNav.style.display = 'none';
+  defaultNav.style.display = 'unset';
+}
+
+hamburger.addEventListener('click', displayMobileMenu);
+xButton.addEventListener('click', hideMobileMenu);
```

Comment:
Well done overall 💯 but one of the requirements is that when the user clicks (or taps) any option of the mobile menu, the mobile menu should disappear. Yours is not disappearing for now. 😶‍🌫️

Reply:
add an event listener to hide the mobile menu when the user clicks the option

Reply:
hide the mobile menu is when the user click is DONE.